presets/imcs/workstation/CMakePresets.json

```diff
@@ -19,7 +19,8 @@
         "FOUR_C_BUILD_DOXYGEN": "ON",
         "FOUR_C_WITH_GOOGLETEST": "ON",
         "FOUR_C_WITH_ARBORX": "ON",
-        "FOUR_C_WITH_MIRCO": "ON"
+        "FOUR_C_WITH_MIRCO": "ON",
+        "FOUR_C_BUILD_SHARED_LIBS": "OFF"
```

Comment:
You are, of course, free to do this, but I would be interested if there is a specific reason you like static libs here.